$NetBSD: patch-aa,v 1.1 2011/01/06 16:36:32 thomasklausner Exp $

Avoid unnecessary build-breaking warning about uninitialized variable
with mono-2.8.1.

--- src/PdfMod/Gui/Client.cs.orig	2010-09-28 19:45:35.000000000 +0000
+++ src/PdfMod/Gui/Client.cs
@@ -415,7 +415,7 @@ namespace PdfMod.Gui
         static void OnLogNotify (LogNotifyArgs args)
         {
             ThreadAssist.ProxyToMain (delegate {
-                Gtk.MessageType mtype;
+                Gtk.MessageType mtype = Gtk.MessageType.Error;
                 var entry = args.Entry;
 
                 switch (entry.Type) {
